packages/ffe-accordion-react/src/AccordionItem.spec.js

```diff
@@ -54,27 +62,25 @@ describe('<AccordionItem />', () => {
             }); // esc
             expect(wrapper.state('isOpen')).toBe(false);
         });
-        it('toggling triggers onOpen and onClose props', () => {
-            const onOpenSpy = sinon.spy();
```

Comment:
Travis correctly points out `sinon` is no longer used after this, so the import should be removed.

Reply:
Gah my bad! Thanks!